src/oidcendpoint/oidc/token.py

```diff
@@ -83,26 +115,89 @@ def _access_token(self, req, **kwargs):
         issue_refresh = False
         if "issue_refresh" in kwargs:
             issue_refresh = kwargs["issue_refresh"]
+        else:
+            if "offline_access" in grant.scope:
+                issue_refresh = True
+
+        _response = {
+            "token_type": "Bearer",
+            "expires_in": 900,
```

Comment:
This is still hardcoded, this should be the access token's lifetime

Reply:
I've changed this.